fgpyo/util/tests/test_metric.py

```diff
@@ -171,8 +172,8 @@ def test_metric_parse() -> None:
     assert Person.parse(fields=["name", "42"]) == Person(name="name", age=42)
 
 
-# def test_metric_parse_with_None() -> None:
-#    assert Person.parse(fields=[None, "40"]) == Person(name='', age=40)
+def test_metric_parse_with_None() -> None:
+    assert Person.parse(fields=["", "40"]) == Person(name="None", age=40)
```

Comment:
Wait, don't we want name to be `None` not `"None"`.  I.e. the value `None`, not the string with contents `"None"`?  I am not sure how tests are passing since I _think_ your code in `metric.py` is correct.
```suggestion
    assert Person.parse(fields=["", "40"]) == Person(name=None, age=40)
```

Reply:
Add a unit test for testing `format_value` too